### PR TITLE
Tokens - Removes Old Tokens

### PR DESCRIPTION
**Pull request by anechol**
# Description
Removes the old tokens set from the repo and cleans up small token issues in some components.

[DSS-516](https://kajabi.atlassian.net/browse/DSS-516)

## Type of change

- [x] Chore (removes unused code)

# How Has This Been Tested?

Check components to ensure that no styles are missing or incorrect.

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [x] tested manually
- [ ] other:

**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

[DSS-516]: https://kajabi.atlassian.net/browse/DSS-516?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/core/scripts/sd-transforms.mjs [hunk 1]
```diff
@@ -21,6 +21,9 @@ const getConfig = (sets) => {
 					destination: `${tokenSet}/_${tokenSet}.scss`,
 					format: 'css/variables'
 				})),
+				options: {
+					showFileHeader: false
+				},
```
Comment (ju-Skinner) [inline libs/core/scripts/sd-transforms.mjs:26]: ❤️

File: libs/core/src/components/pds-button/stories/pds-button.stories.js [hunk 2]
```diff
@@ -54,6 +54,7 @@ Primary.args = {
   disabled: false,
   slot: 'Primary',
   type: 'button',
+  variant: 'primary'
```
Comment (QuintonJason) [inline libs/core/src/components/pds-button/stories/pds-button.stories.js:57]: 🔬

File: libs/core/src/components/pds-input/pds-input.scss [hunk 3]
```diff
@@ -2,7 +2,7 @@
   --box-shadow-focus: 0 0 0 2px var(--pine-color-blue-200);
   --box-shadow-focus-error: 0 0 0 2px var(--pine-color-red-200);
 
-  --color-text-default: var(-pine-color-charcoal-500);
+  --color-text-default: var(--pine-color-charcoal-500);
```
Comment (QuintonJason) [inline libs/core/src/components/pds-input/pds-input.scss:5]: 🔬

**Review by QuintonJason — APPROVED** [review]
Good to go

**Review by ju-Skinner — APPROVED** [review]

**Pull request merged**